leader-election-api/src/main/java/com/palantir/paxos/PaxosProposer.java

```diff
@@ -42,10 +45,12 @@
      * {@link PaxosProposer#propose(long, byte[])}. However, this value is proposed anonymously, with a fresh
      * proposer ID. This may be useful for relinquishing leadership in algorithms for that based on Paxos.
      */
+    @Timed
     byte[] proposeAnonymously(long seq, @Nullable byte[] proposalValue) throws PaxosRoundFailureException;
 
     /**
      * Returns a unique string identifier for the proposer.
      */
+    @Timed
```

Comment:
this is an in memory read

Reply:
doh, I thought this was a remote call, for some reason. Will remove